### PR TITLE
[64] - Review Fix : Make asserter pay for the bond

### PR DESCRIPTION
**Pull request by 0xNuggan**
Fixed as per comment https://github.com/InverterNetwork/contracts/pull/619#discussion_r1700629388

### CODE REVIEW

File: src/proxies/InverterBeacon_v1.sol [hunk 1]
```diff
@@ -63,10 +63,10 @@ contract InverterBeacon_v1 is IInverterBeacon_v1, ERC165, Ownable2Step {
     ) {
         if (
             // Minor Version cant go down
-            newMinorVersion < minorVersion
             // Patch Version cant go down or stay the same if minorVersion stays the same
-            || newPatchVersion <= patchVersion
-                && newMinorVersion == minorVersion
+            newMinorVersion < minorVersion
+                || newPatchVersion <= patchVersion
+                    && newMinorVersion == minorVersion
```
Comment (marvinkruse) [inline src/proxies/InverterBeacon_v1.sol:69]: What happened here? Your version looks fine, but interesting that the "before" part looks so weird...

**Review by marvinkruse — APPROVED** [review]

File: src/proxies/InverterBeacon_v1.sol (hunk 1 above)
Comment (marvinkruse) [reply src/proxies/InverterBeacon_v1.sol:69]: Oh I see, you just didn't like the formatting, the version before was fine, just had comments within the expression :D Or did you not change this? @0xNuggan

Comment (marvinkruse) [reply src/proxies/InverterBeacon_v1.sol:69]: Oh it formats it the same for me locally after updating my Foundry, ignore this 😂

Comment (0xNuggan) [reply src/proxies/InverterBeacon_v1.sol:69]: That was the formatter. The comments were originally inlined directly inside the if() statement, which probably goes against some rule.

**Pull request merged**